ng-dev/release/notes/release-notes.ts

```diff
@@ -50,6 +60,28 @@ export class ReleaseNotes {
     return render(changelogTemplate, await this.generateRenderContext(), {rmWhitespace: true});
   }
 
+  /** Prepends the generated release note to the CHANGELOG file. */
+  async prependEntryToChangelog() {
+    /** The fully path to the changelog file. */
+    const filePath = join(this.git.baseDir, 'CHANGELOG.md');
```

Comment:
We should have a constant for the `CHANGELOG.md` file, like we do for the release action code. Ideally this constant should be in `release/versioning` as a common place to capture version-specific things (maybe there already is a constant for this; I don't recall)

Reply:
I moved the `CHANGELOG.md` constant into `ng-dev/release/notes`